### ISSUE TITLE
New node joining needs to be updated

### ISSUE THREAD

**Issue by ansnunez** [Zilliqa/Zilliqa#501]
New node joining has not been updated after PoW merge work.
In current master, it still performs PoW2.  Simply changing it to use PoW (#500) doesn't work either.

In either case (using PoW2 or PoW), the following warning always appears after attempting PoW:

```
[INFO   ][TID    75][06:23:01][SetState                      ][Epoch 25] Node State is now POW_SUBMISSION at epoch 25
[INFO   ][TID    75][06:23:06][StartPoW                      ] BEGIN
[INFO   ][TID    75][06:23:06][StartPoW                      ][Epoch 25] Current dsblock is 6
[INFO   ][TID    75][06:23:06][PoWMine                       ] BEGIN
[WARNING][TID    75][06:23:06][EthashConfigureLightClient    ] WARNING: How come the latest block number is smaller than current block number.  block_number:  currentBlockNum: 7
```

**Comment by bb111189** [comment]
New node previous joined in at PoW2. With the removal of PoW2, new node need to join at PoW1 (also known as pow now) instead. 

Previously, in PoW2, inputs to PoW is rand1 and rand2 where rand2 is set to 0x0. With the merging, node need to do PoW1 instead. Hence, a new node need to 

- updateTxBlkRand() 
- set rand2 to TxBlkRand

**Comment by ansnunez** [comment]
This issue has been addressed in #507
